docs/standard/datetime/snippets/timeprovider-overview/vb/myproject.vbproj

```
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>dateonlytimeonly</RootNamespace>
+    <TargetFramework>net8.0</TargetFramework>
```

Is `net9.0` not available for VB? 😛 

```suggestion
    <TargetFramework>net9.0</TargetFramework>
```